static-build/qemu-virtiofs/Dockerfile

```diff
@@ -38,6 +38,8 @@ RUN apt-get install -y \
 
 RUN cd  .. && git clone "${QEMU_VIRTIOFS_REPO}" qemu-virtiofs
 RUN git checkout "${QEMU_VIRTIOFS_TAG}"
+ADD qemu/patches/virtiofsd/0001-add-time-to-seccomp.patch /root/0001-add-time-to-seccomp.patch
+RUN patch -p1 < /root/0001-add-time-to-seccomp.patch
```

Comment:
this is a signed patch, why don't apply it with `git am` ?

Reply:
I can do it with `git am`, but will need to add steps to configure git user and email, wdyt? Should we go with `git am`?

Reply:
ohh I see, then just `patch` is fine